rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Getters and Setters for @book and @person instance variables should be present
```suggestion
  attr_accessor :date, :book, :person
```